docs-app/.template-lintrc.js

```diff
@@ -2,4 +2,13 @@
 
 module.exports = {
   extends: 'recommended',
+
+  overrides: [
+    {
+      files: ['**/*.gjs', '**/*.gts'],
+      rules: {
+        'no-implicit-this': false,
+      },
+    },
+  ],
```

Comment:
This shouldn't be needed due to https://github.com/ember-template-lint/ember-template-lint/pull/2844.  Going to dig in more after a bit.